std_msgs/Header header

## pos
float32[3] position         ## [m]
float32[3] velocity         ## [m/s]
float32[3] angular_velocity ## [rad/s]
float32[3] Euler            ## [rad]
float32[4] quaternion        ##
